apps/frontend/src/utilities/splunk_util.ts

```diff
@@ -235,15 +231,17 @@ export class SplunkEndpoint {
   }
 
   /** Returns the current state of the job */
-  private async check_job(job_id: JobID): Promise<JobState> {
-    return fetch(`${this.host}/services/search/jobs/${job_id}`, {
+  private async check_job(jobId: JobID): Promise<JobState> {
+    return fetch(`${this.host}/services/search/jobs/${jobId}`, {
       method: 'GET',
       headers: new Headers({
-        Authorization: this.auth_string
+        Authorization: this.authString
       })
     })
       .then((response) => {
-        if (!response.ok) throw process_error(response);
+        if (!response.ok) {
+          throw process_error(response);
+        }
         return response.text();
       })
```

Comment:
`(response) => { <function body }` is just a declaration of an inline [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions). Since this function is used twice (here and below), it should just be converted to a real function that does the exact same thing. The `.then` method just takes a function,  so you should be able to say `.then(process_response)` which will call a function called `process_response` which takes a `response` parameter.